specs/src/shrex/shrex-sub.md

```diff
@@ -0,0 +1,178 @@
+# ShrEx/Sub Protocol Specification
+
+## Abstract
+
+**ShrEx/Sub** (Share Exchange/Subscribe) is a push-based notification protocol implementing the publish-subscribe pattern for Extended Data Square (EDS) hash distribution in the Celestia data availability network. The protocol enables efficient dissemination of new EDS availability notifications across different node types.
```

Comment:
I wish we had a place to point to what EDS is

Reply:
I think this is a bit misleading to say that it distributes eds notifications. It is just new blocks being available. Notification mainly carries the height and eds root hash only to verify notification